plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Drilldown.js

```diff
@@ -273,6 +274,15 @@ Ext.define('NX.view.drilldown.Drilldown', {
           }
         });
       }
+
+      // Clear successive filters
+      if (i > index) {
```

Comment:
might as well put this in the same if block as above, make it clear that both are triggered by the same condition